engine/graphics/proto/graphics/graphics_ddf.proto

```diff
@@ -274,6 +274,7 @@ message PlatformProfile
         OS_ID_SWITCH  = 7;
         OS_ID_PS4     = 8;
         OS_ID_PS5     = 9;
+        OS_ID_XBOX    = 10;
```

Comment:
I'm not a super fan of having the platforms listed here tbh.
They're part of the texture profiles api, so it'll be there for a while, but still not very data driven...